telethon/telegram_client.py

```diff
@@ -1779,7 +1779,12 @@ def _on_handler(self, update):
             event = builder.build(update)
             if event:
                 event._client = self
-                callback(event)
+                try:
+                    callback(event)
+                except StopPropagation:
+                    __log__.info("Event handler '{}' stopped chain of propagation for update {}.".format(
```

Comment:
Whoops, longer than 80 characters :/

Reply:
btw, y u no 100 chars width?

Reply:
I just settled with 80 and try to stick to it. It's comfortable for `git diff` (yes I can resize the terminal or even change its default size).